src/lib_c/x86_64-portbld-freebsd/c/sys/types.cr

```diff
@@ -9,9 +9,17 @@ lib LibC
   alias DevT = UInt
   alias GidT = UInt
   alias IdT = Long
-  alias InoT = UInt
+  {% if flag?(:"freebsd12.0") %}
```

Comment:
No need to use symbol, you can just write:

```crystal
{% if flag?("freebsd12.0") %}
```

Reply:
Hm, now this was merged with symbol...?

Reply:
It was just a note, not super necessary to make it work. There's also no difference regarding performance or anything, just style.